packages/template-retail-react-app/app/hooks/use-auth-modal.js

```diff
@@ -294,6 +296,8 @@ export const AuthModal = ({
                             form={form}
                             submitForm={submitForm}
                             clickCreateAccount={() => setCurrentView(REGISTER_VIEW)}
+                            clickForgotPassword={() => setCurrentView(PASSWORD_VIEW)}
```

Comment:
this `clickForgotPassword` handler was renamed to `handleForgotPasswordClick` below as Ben mentioned handler functions should be prefixed with `handle` for consistency